packages/core/package.json

```diff
@@ -92,6 +92,7 @@
     "camelcase": "^6.2.0",
     "cross-env": "^7.0.3",
     "eslint": "^8.6.0",
+    "framer-motion": "^6.5.1",
```

Comment:
@pcln-james I'm not sure if I'm allowed to add this but the `Animate` component imports it so I did 🤔

Reply:
I didn't realize Animate wasn't on v4, you can remove this. the v4 Toasts will just be less fancy

Reply:
we can remove Animate from Toast and the addition of it for this PR

Reply:
removed